src/conn/conn_cache.c

```diff
@@ -70,76 +186,23 @@ __cache_config_local(WT_SESSION_IMPL *session, bool shared, const char *cfg[])
         conn->cache_size = (uint64_t)cval.val;
     }
 
+    /* Set config values as percentages. */
     WT_RET(__wt_config_gets(session, cfg, "cache_overhead", &cval));
     cache->overhead_pct = (u_int)cval.val;
-
-    WT_RET(__wt_config_gets(session, cfg, "eviction_target", &cval));
-    cache->eviction_target = (double)cval.val;
     WT_RET(
       __cache_config_abs_to_pct(session, &(cache->eviction_target), "eviction target", shared));
```

Comment:
Thanks for picking this up, I've modified `__validate_cache_config` so now the values gets changed to percentage straight away after retrieving and setting the value. See this commit: [9e4baa7](https://github.com/wiredtiger/wiredtiger/pull/10548/commits/9e4baa7696ed61b4507e4766121155606277fcd8)
I'd prefer to change the way we access `conn->cache` in another ticket because I think there's already quite a bit of changes going on for this PR, it might be clearer to do for a seperate ticket.

Reply:
SGTM! I've created WT-13161 to track this